src/amo/components/HeroRecommendation/styles.scss

```diff
@@ -0,0 +1,173 @@
+@import '~amo/css/styles';
+
+.HeroRecommendation {
+  color: $white-100;
+  display: flex;
+  padding: 24px;
+  position: relative;
+
+  @include respond-to(extraExtraLarge) {
+    padding: 36px;
+
+    @include padding-start(96px);
+  }
+}
+
+.HeroRecommendation-info {
+  @include margin-start(24px);
+
+  @include respond-to(extraLarge) {
+    @include margin-start(96px);
+  }
+
+  @include respond-to(extraExtraLarge) {
+    @include margin-start(136px);
+  }
+}
+
+$overlayShapeZIndex: 1;
+
+.HeroRecommendation-overlayShape {
+  bottom: 0;
+  height: 70%;
+  left: 0;
+  max-height: 307px;
+  position: absolute;
+  z-index: $overlayShapeZIndex;
+
+  [dir='rtl'] & {
+    left: auto;
+    right: 0;
+    transform: scaleX(-1);
+  }
+}
+
+.HeroRecommendation-image {
+  height: auto;
+  width: 100%;
+
+  @include respond-to(extraExtraLarge) {
+    width: 480px;
+  }
+}
+
+.HeroRecommendation-recommended {
+  font-size: $font-size-s;
+  letter-spacing: 0.1em;
+  line-height: 1.143;
+  margin: 0;
+  opacity: 0.5;
+  text-transform: uppercase;
```

Comment:
I thought we were trying not to use this transformation because it is bad for accessibility? https://github.com/mozilla/addons-frontend/issues/3322#issuecomment-333957730

Reply:
From [the same linked article](http://uxmovement.com/content/all-caps-hard-for-users-to-read/):

> When is it okay to use all caps? All caps are fine in contexts that don’t involve much reading, such as logos, headings, acronyms, and abbreviations.

So, I think it's fine in this context. The login bug you cited was for a case where the text definitely needed to be readable. I will double check with Emanuela to make sure it was her intent.

Reply:
Emanuela agreed that it's fine but she suggested making the L10n string uppercase so that localizers have a chance to decide to upper case it or not, based on locale.

Reply:
Ok, great.